### PR TITLE
Address review comments on lesson 12

### LINKED ISSUE #180: Review comments on lesson 12
- [x] "HPC system" definition should use the word "integrated"?
      Maybe in contrast to "independent" for typical scenarios.
- [x] For some systems, you won't be asked for your password.
- [x] Who is Lola?
- [x] Windows SSH discussion is not the same as that in the setup.
- [x] Head nodes are also sometimes called landing pads.
- [x] "the scheduler used in this lesson is [missing]"
- [x] Shared file sytem callout could use some intro/context.
- [x] Near the df output is maybe better?
- [x] "All of a HPC ...", wording.
- [x] Node description could mention storage can be local or shared.
- [x] "Explore your computer" requires being on the local machine
      but the instruction flow is still on the head node.
- [x] "Units and language" callout could be moved to the glossary.
---
- [x] Capitalization on PuTTY

### PR DESCRIPTION
**Pull request by tkphd**
Small set of relatively minor changes. Closes #180.

### CODE REVIEW

File: _episodes/12-cluster.md [hunk 1]
```diff
@@ -62,27 +62,14 @@ Go ahead and log in to the cluster: {{ site.remote.name }} at {{ site.remote.loc
...
-Remember to replace `yourUsername` with the username supplied by the instructors. You will be asked
-for your password. But watch out, the characters you type are not displayed on the screen.
+Remember to replace `yourUsername` with the username supplied by the instructors. You may be asked
+for your password. Watch out: the characters you type are not displayed on the screen.
 
 You are logging in using a program known as the secure shell or `ssh`. This establishes a temporary
 encrypted connection between your laptop and `{{ site.remote.login }}`. The word before the `@`
-symbol, e.g. `yourUsername` here, is the user account name that Lola has access permissions for on
-the cluster.
+symbol, e.g. `yourUsername` here, is the user account name that you permission to use on the
```
Comment (reid-a) [inline _episodes/12-cluster.md:70]: There's a grammatical issue with "you permission to use..." here.

File: _episodes/12-cluster.md [hunk 2]
```diff
@@ -62,27 +62,14 @@ Go ahead and log in to the cluster: {{ site.remote.name }} at {{ site.remote.loc
 ```
 {: .bash}
 
-Remember to replace `yourUsername` with the username supplied by the instructors. You will be asked
-for your password. But watch out, the characters you type are not displayed on the screen.
+Remember to replace `yourUsername` with the username supplied by the instructors. You may be asked
+for your password. Watch out: the characters you type are not displayed on the screen.
```
Comment (reid-a) [inline _episodes/12-cluster.md:66]: The characters you type *at* *the* *password* *prompt* are not displayed on the screen.

File: _episodes/12-cluster.md (hunk 1 above)
Comment (tkphd) [reply _episodes/12-cluster.md]: f14c07b

File: _episodes/12-cluster.md (hunk 2 above)
Comment (tkphd) [reply _episodes/12-cluster.md]: f14c07b

File: _episodes/12-cluster.md [hunk 3]
```diff
@@ -162,19 +135,18 @@ notice that the current hostname is also part of our prompt!)
 Individual computers that compose a cluster are typically called *nodes* (although you will also
 hear people call them *servers*, *computers* and *machines*). On a cluster, there are different
 types of nodes for different types of tasks. The node where you are right now is called the *head
-node*, *login node* or *submit node*. A login node serves as an access point to the cluster. As a
-gateway, it is well suited for uploading and downloading files, setting up software, and running
-quick tests. It should never be used for doing actual work.
+node*, *login node*, *landing pad*, or *submit node*. A login node serves as an access point to the
+cluster. As a gateway, it is well suited for uploading and downloading files, setting up software,
```
Comment (psteinb) [inline _episodes/12-cluster.md:139]: `As a gateway, it is well suited for uploading and downloading files` not sure this is true while reading it. I propose:

```suggestion
cluster. As a gateway, it is well suited for uploading and downloading small files, setting up software,
```

Shall we fix it right now?

Comment (tkphd) [reply _episodes/12-cluster.md:139]: I don't think the "small" qualifier helps to clarify anything; if anything, it opens a discussion, "how small is small," that would be distracting. Practically, the limit is your filesystem quota, and whether your bandwidth utilization impacts other users.

We could open a separate issue to call out data transfer nodes, which are sometimes available for large (high-bandwidth) data transfers?

Comment (psteinb) [reply _episodes/12-cluster.md:139]: I like the idea with a call-out on this!

**Pull request merged**